Adds 11x Case Study (#11043)
PostHog/posthog.com: contents/customers/11x.md

---
title: How 11x uses PostHog to combat legal threats, build AI workers 
customer: 11x
logo: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/11x_logo_light_8c7d326edb.png
logoDark: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/11x_logo_dark_0934407584.png
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/11x_720_ec829e8b81.jpg
industries:
  - AI
users:
  - Marketing
  - Leadership
  - Founder
  - Engineering
toolsUsed:
  - Experimentation
  - Product analytics
  - LLM analytics
  - CDP
date: 2023-09-15
---

Trusted by the likes of PWC, 11x is an AI and automation company building the next generation of fully autonomous digital workers, such as [Alice](https://www.11x.ai/worker/alice) a virtual SDR for rapidly growing companies.

> As part of a new wave of AI-first teams, 11x is creating cutting-edge digital agents which are can integrate fully with your entire stack  — but an AI-first company relies on data just like any other SaaS. So, the team formed a complex stack of tools to help them ensure the AI was performing well. 

Reviewer: ```suggestion
As part of a new wave of AI-first teams, 11x is creating cutting-edge digital agents that can integrate with your entire stack. But an AI-first company relies on data just like any other SaaS, so the team formed a complex stack of tools to help them ensure the AI was performing well. 
```

“It was a real hodgepodge of tools,” says Keith Fearon, Head of Growth at 11x. “We used [HotJar](/blog/posthog-vs-hotjar) for session recordings, [FullStory](/blog/posthog-vs-fullstory) for analytics, and we were building our own in-house feature flags.”

> Eventually, after two years, the team decided to optimize things — and soon discovered that PostHog offered all of these tools in one platform, complete with its own [dedicated AI analytics suite](/docs/ai-engineering)

Reviewer: ```suggestion
Eventually, after two years, the team decided to optimize things. They soon discovered that PostHog offered all of these tools in one platform, complete with its own [dedicated AI analytics suite](/docs/ai-engineering)
```
